build-system/server/lazy-build.js

```diff
@@ -131,6 +144,8 @@ async function lazyBuildJs(req, _res, next) {
  * Pre-builds the core runtime and the JS files that it loads.
  */
 async function preBuildRuntimeFiles() {
+  esbuildWatchedExtensions.add('amp.js');
+  esbuildWatchedExtensions.add('ww.max.js');
```

Comment:
This seems like a detail of implementation that `preBuildRuntimeFiles()` shouldn't know anything about. Will suggest a change to move this into `build()`.

Reply:
good thinking! I'll make this change now